src/lib/Containers/__tests__/__snapshots__/Gene-tests.tsx.snap

```diff
@@ -34,38 +34,19 @@ exports[`looks like expected 1`] = `
     }
   >
     <ArtworksGrid
-      gene={
+      filtered_artworks={
         Object {
-          "_id": "a UUID",
```

Comment:
should this be worrying? The artwork metadata was remove

Reply:
This is what it looks like:

![simulator screen shot 12 jan 2018 16 45 20](https://user-images.githubusercontent.com/2320/34882822-0a21bf08-f7b8-11e7-9c54-1ce793b8c5ef.png)